drivers/unix/net_socket_posix.cpp

```diff
@@ -334,7 +334,7 @@ Error NetSocketPosix::connect_to_host(IP_Address p_host, uint16_t p_port) {
 	struct sockaddr_storage addr;
 	size_t addr_size = _set_addr_storage(&addr, p_host, p_port, _ip_type);
 
-	if (::connect(_sock, (struct sockaddr *)&addr, addr_size) == SOCK_EMPTY) {
+	if (::connect(_sock, (struct sockaddr *)&addr, addr_size) == (int)SOCK_EMPTY) {
```

Comment:
Same as above, make this `::connect(...) != 0`.

Reply:
Fixed.